docs/sphinx/developers/tiling.txt

```diff
@@ -0,0 +1,129 @@
+Tiled reading and writing in Bio-Formats
+========================================
+
+This guide pertains to version 5.3 and later.
```

Comment:
Not sure this adds much given that it's part of the 5.3 docs bundle (and surely things might change "later") and tiling will get a mention in "what's new".